api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -688,20 +712,20 @@ def test_github_webhook_with_non_existing_installation(
     # When
     response = api_client.post(
         path=url,
-        data=WEBHOOK_PAYLOAD_WITH_AN_INVALID_INSTALLATION_ID,
+        data=WEBHOOK_PAYLOAD_WITHOUT_INSTALLATION_ID,
         content_type="application/json",
-        HTTP_X_HUB_SIGNATURE=WEBHOOK_SIGNATURE_WITH_AN_INVALID_INSTALLATION_ID,
+        HTTP_X_HUB_SIGNATURE=WEBHOOK_SIGNATURE_WITHOUT_INSTALLATION_ID,
         HTTP_X_GITHUB_EVENT="installation",
     )
 
     # Then
     mocker_logger.error.assert_called_once_with(
-        "GitHub Configuration with installation_id 765432 does not exist"
+        "The installation_id is not present in the payload: {'installation': {'test': 765432}, 'action': 'deleted'}"
     )
     assert response.status_code == status.HTTP_200_OK
```

Comment:
Should this error really return an HTTP 200 response?

Reply:
In this case, 200 is enough.